### ISSUE TITLE
Cannot close modal by  MicroModal.close(modal.id);

### ISSUE THREAD

**Issue by srikanthcx** [ghosh/micromodal#133]
cannot close or open modal by using Micromodal module in React 

MicroModal.close(modal.id);

**Comment by donbot-inc** [comment]
Same problem:
```
Uncaught TypeError: Cannot read property 'closeModal' of null
    at Object.close (micromodal.js:343)
```

**Comment by kgoedecke** [comment]
Workaround if you use jQuery:
```
$('#modal-1').removeClass('is-open');
```

**Comment by vonPiernik** [comment]
> Workaround if you use jQuery:
> 
> ```
> $('#modal-1').removeClass('is-open');
> ```
> modal-1 is your modal ID.

This does not change aria-hidden to true so may cause accessibility issues. @djonatanb solution with using show() works ;)

**Comment by ghosh** [comment]
Fixed in #113

**Comment by VitaliiThir** [comment]
onShow: () => {
      setTimeout(function () {
        $('[data-micromodal-close]').trigger('click');
      }, 5000)
    }
